pkg/internalapis/cnsvolumeinfo/cnsvolumeinfoservice.go

```diff
@@ -184,6 +189,43 @@ func (volumeInfo *volumeInfo) CreateVolumeInfo(ctx context.Context, volumeID str
 	return nil
 }
 
+// CreateVolumeInfo creates VolumeInfo CR to persist VolumeID to vCenter mapping
+func (volumeInfo *volumeInfo) CreateVolumeInfoWithPolicyInfo(ctx context.Context,
+	volumeID string, storagePolicyId string, storageClassName string, vCenter string) error {
+	log := logger.GetLogger(ctx)
+	log.Infof("creating cnsvolumeinfo for volumeID: %q, StoragePolicyID: %q, "+
+		"StorageClassName: %q, vCenter: %q in the namespace: %q",
+		volumeID, storagePolicyId, storageClassName, vCenter, csiNamespace)
+
+	volumeInfoCrName := getCnsColumeInfoCrName(ctx, volumeID)
```

Comment:
fix typo :  getCnsColumeInfoCrName ->  getCnsVolumeInfoCrName

Reply:
I'm leaving this unchanged as the `getCnsVolumeInfoCrName` was added before this PR.
But, I will address it in the subsequent PR